.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,110 @@
+name: ci
+on: [push, pull_request]
+env:
+  CI: true
+
+jobs:
+  ubuntu-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies for Ubuntu
+      run: |
+        sudo apt-get install neofetch
+        # Install PyTorch for Linux with no CUDA support
+        pip install torch==1.4.0+cpu torchvision==0.5.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
+    - name: Print Version Info
+      run: |
+        neofetch
+        python --version
+        pip --version
+    - name: Install Emmental
+      run: |
+        make dev
+        pip install -q coveralls
+    - name: Run preliminary checks
+      run: |
+        make check
+        make docs
+    - name: Test with pytest
+      run: |
+        coverage run -m pytest tests
+    - name: Upload to coveralls.io
+      run: |
+          coveralls
```

Comment:
You probably want to call combine here, like with Fonduer, I'd assume.

Reply:
`coverage combine` gave me this error.
```
No data to combine
##[error]Process completed with exit code 1.```